airflow/providers/databricks/triggers/databricks.py

```diff
@@ -32,14 +32,30 @@ class DatabricksExecutionTrigger(BaseTrigger):
     :param databricks_conn_id: Reference to the :ref:`Databricks connection <howto/connection:databricks>`.
     :param polling_period_seconds: Controls the rate of the poll for the result of this run.
         By default, the trigger will poll every 30 seconds.
+    :param retry_limit: The number of times to retry the connection in case of service outages.
+    :param retry_delay: The number of seconds to wait between retries.
+    :param run_page_url: The run page url.
     """
 
-    def __init__(self, run_id: int, databricks_conn_id: str, polling_period_seconds: int = 30) -> None:
+    def __init__(
+        self,
+        run_id: int,
+        databricks_conn_id: str,
+        polling_period_seconds: int = 30,
+        retry_limit: int = 3,
+        retry_delay: int = 10,
+        run_page_url: str | None = None,
```

Comment:
Please unify it with sync version of operator - it allows to have more sophisticated retries, like this: https://github.com/apache/airflow/blob/main/airflow/providers/databricks/operators/databricks.py#L305.

Reply:
Added `databricks_retry_args` too in hook initialization in the trigger